src/util/ChartUtils.ts

```diff
@@ -668,31 +668,31 @@ export const getTicksOfAxis = (
 /**
  * combine the handlers
  * @param  {Function} defaultHandler Internal private handler
- * @param  {Function} parentHandler  Handler function specified in parent component
- * @param  {Function} childHandler   Handler function specified in child component
+ * @param  {Function} childHandler Handler function specified in child component
  * @return {Function}                The combined handler
  */
-export const combineEventHandlers = (defaultHandler: Function, parentHandler: Function, childHandler: Function) => {
-  let customizedHandler: Function;
+const handlerWeakMap = new WeakMap();
```

Comment:
I like this change but how has this been tested? Can we add unit tests?

Reply:
I'll try to add it as time allows.